Source/WebCore/Modules/webcodecs/WebCodecsVideoEncoder.cpp

```diff
@@ -147,22 +167,16 @@ ExceptionOr<void> WebCodecsVideoEncoder::configure(ScriptExecutionContext& conte
             };
         }
 
-        auto encoderConfig = createVideoEncoderConfig(config);
-        if (encoderConfig.hasException()) {
-            closeEncoder(Exception { NotSupportedError, encoderConfig.releaseException().message() });
-            return;
-        }
-
         VideoEncoder::create(config.codec, encoderConfig.releaseReturnValue(), [this, weakThis = WeakPtr { *this }](auto&& result) {
             if (!weakThis)
                 return;
 
+            m_isMessageQueueBlocked = false;
```

Comment:
this doesn't seem related to the change being described. Can the commit log include a note?

Reply:
I updated the commit message.